packages/flame_tiled/example/.gitignore

```diff
@@ -0,0 +1,43 @@
+# Miscellaneous
```

Comment:
This gitignore shouldn't be needed, there is already one in the root of the repo that should be covering everything.